oauth2-authorization.js

```diff
@@ -110,6 +127,9 @@ export class OAuth2Authorization extends HTMLElement {
       case 'authorization_code':
         this._authorize(this._constructPopupUrl(settings, 'code'), settings);
         break;
+      case 'refresh_token':
```

Comment:
Why do we need `refresh_token`? It does exactly the same thing as `authorization_code`. If so I would not put it in here.